src/components/pages/auth/register/index.tsx

```diff
@@ -55,6 +55,7 @@ export default function RegisterComponent() {
       setRegisterStatus("success");
       setIsLoading(false);
       toastify("Register Successful. Please check your email to verify", "success");
+      router.replace("/feed");
```

Comment:
We don't want to send the user to the /feed page

Reply:
okay what about sending it to login page instead